browser/brave_shields/brave_shields_web_contents_observer.cc

```diff
@@ -58,18 +61,49 @@ BraveShieldsWebContentsObserver::BraveShieldsWebContentsObserver(
       receivers_(web_contents, this) {}
 
 void BraveShieldsWebContentsObserver::RenderFrameCreated(RenderFrameHost* rfh) {
-  if (rfh && allowed_scripts_.size()) {
-    GetBraveShieldsRemote(rfh)->SetAllowScriptsFromOriginsOnce(
-        allowed_scripts_);
-  }
-  if (rfh) {
-    if (content::BrowserContext* context = rfh->GetBrowserContext()) {
-      if (PrefService* pref_service = user_prefs::UserPrefs::Get(context)) {
-        GetBraveShieldsRemote(rfh)->SetReduceLanguageEnabled(
-            brave_shields::IsReduceLanguageEnabledForProfile(pref_service));
-      }
+  SendShieldsSettingsToFrame(rfh, nullptr);
```

Comment:
Do we really need this taking into account we handle `ReadyToCommitNavigation()` ?

Reply:
P.S. there is per-RFH structure called [DocumentUserData](https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/document_user_data.h;l=98) that allows to avoid manual frame tracking.

Reply:
I wanted to keep most things without changes and then do a separate cleanup, but I guess it's fine to cleanup some obvious things right now.